java-checks/src/main/java/org/sonar/java/checks/tests/AbstractJUnit5NotCompliantModifierChecker.java

```diff
@@ -42,7 +46,15 @@ public abstract class AbstractJUnit5NotCompliantModifierChecker extends Issuable
   protected abstract void raiseIssueOnNonCompliantReturnType(MethodTree methodTree);
 
   protected void raiseIssueOnNonCompliantModifier(ModifierKeywordTree modifier) {
-    reportIssue(modifier, String.format(WRONG_MODIFIER_ISSUE_MESSAGE, modifier.keyword().text()));
+    QuickFixHelper.newIssue(context)
+      .forRule(this)
+      .onTree(modifier)
+      .withMessage(String.format(WRONG_MODIFIER_ISSUE_MESSAGE, modifier.keyword().text()))
+      .withQuickFixes(() -> List.of(
```

Comment:
You can use `.withQuickFix` (singular) instead of `.withQuickFixes` and pass it the single `JavaQuickFix` created, instead of instantiating a `List` 

```suggestion
     .withQuickFix(() -> 
        JavaQuickFix.newQuickFix("Remove modifier")
        .addTextEdit(JavaTextEdit.removeTree(modifier))
        .build())